### PR TITLE
bookmark tooltip enhancement

### PR DESCRIPTION
**Pull request by vfyjxf**
Enhanced tooltip for recipe bookmark

### CODE REVIEW

File: Forge/src/main/java/mezz/jei/forge/platform/RenderHelper.java [hunk 1]
```diff
@@ -79,4 +81,11 @@ public void renderTooltip(
 			y
 		);
 	}
+
+	@Override
+	public void renderTooltip(Screen screen, GuiGraphics guiGraphics, List<ClientTooltipComponent> components, int x, int y, Font font, ItemStack stack) {
+//		guiGraphics.tooltipStack = stack;
```
Comment (vfyjxf) [inline Forge/src/main/java/mezz/jei/forge/platform/RenderHelper.java]: I'm not sure what caused AT to not work on my computer, so I commented it here.

**Comment by vfyjxf** [comment]
preview and ingredients list: 
QQ_1720371269858
preview only: 
QQ_1720371302002
To display the ingredients list you need to press shift.

**Comment by vfyjxf** [comment]
QQ_1720466213688
Also, I added error highlight

**Comment by mezz** [comment]
I have hidden this behind SHIFT by default, and made a configuration option so it can always be shown if you want instead.

**Comment by mezz** [comment]
I set the default config to show only the recipe, not the ingredient summary.
I think that will be the most common and useful option for most people.

**Comment by mezz** [comment]
Thank you for your work on this and contributing a big feature idea to JEI!

**Pull request merged**